sopel_waifu/waifu.json

```
@@ -1885,6 +1750,180 @@
         "Isurugi Noe",
         "Andou Aiko"
     ],
+    "Type Moon": [
+        "Abigail Williams",
+        "Altera",
+        "Anastasia Nikolaevna Romanova",
+        "Anne Bonny",
+        "Aozaki Aoko",
+        "Aozaki Touko",
+        "Aphrodite",
+        "Arceuid Brunestud",
+        "Artemis",
+        "Artoria \"Saber\" Pendragon",
+        "Artoria Pendragon (Alter)",
+        "Artoria Pendragon (Lily)",
+        "Asako \"Hassan of the Hundred Persona\"",
+        "Asagami Fujino",
+        "Astraea",
+        "Atalante",
+        "Atalante (Alter)",
+        "Athena",
+        "Bazett Fraga McRemitz",
+        "BB",
+        "Beni-enma",
+        "Boudica",
+        "Bradamante",
+        "Brynhild",
+        "Caenis",
+        "Caren Hortensia",
+        "Calamity Jane",
+        "Carmilla",
+        "Chacha",
+        "Charlotte Corday",
+        "Chevalier d'Eon",
+        "Chloe von Einzbern",
+        "Circe \"Caster of Okeanos\"",
+        "Ciel",
+        "Cleopatra",
+        "Demeter",
+        "Dioscuri Pollux",
+        "Elisabeth Bathory",
+        "Ereshkigal \"Goddess of the Underworld\"",
+        "Europa",
+        "Euryale",
+        "False Assassin",
+        "Fillia",
+        "Francis Drake",
+        "Frankenstein",
+        "Fujimura Taiga",
+        "Gareth",
+        "Gorgon",
+        "Gray",
+        "Helena Blavatsky",
+        "Hera",
+        "Hestia",
+        "Hilde",
+        "Himiko",
+        "Hippolyta",
+        "Hisau Maiya",
+        "Hisui",
+        "Ibaraki-Douji",
+        "Illyasviel \"Ilya\" von Einzbern",
+        "Irisviel \"Iri\" von Einzbern",
+        "Ishtar \"Goddess of Love and War\"",
+        "Jack the Ripper",
+        "Jaguar Warrior",
+        "Jeanne d'Arc",
+        "Jeanne d'Arc (Alter)",
+        "Jeanne d'Arc Alter Santa Lily",
+        "Jinako Carigiri",
+        "Jing Ke",
+        "Kama",
+        "Katou Danzo",
+        "Katsushika Hokusai",
+        "Kijyo Koyo \"Momiji\"",
+        "Kingprotea",
+        "Kiyohime",
+        "Kohaku",
+        "Kokutou Azaka",
+        "Kuonji Alice",
+        "Lakshmi Bai",
+        "Lavinia Whateley",
+        "Leonardo da Vinci",
+        "Leonardo da Vinci (Lily)",
+        "Leysritt",
+        "Luviagelita Edelfelt",
+        "Magical Girl Testament",
+        "Marie Antoinette",
+        "Martha",
+        "Mary Read",
+        "Mash Kirielight",
```

Is this worth tweaking?

```suggestion
        "Mash Kyrielight",
```